### PR TITLE
Adding Nucleo F446ZE FreeRTOS_apps configuration 

### PR DESCRIPTION
**Pull request by alsaibie**
To be reviewed in tandem with [this](https://github.com/micro-ROS/freertos_apps/pull/11) extensions port.

Please **replace** the repo and branch in micro_ros_setup/config/freertos/nucleo_f446ze/board.repos with 
    url: https://github.com/micro-ROS/freertos_apps
    version: dashing
with the merge

### CODE REVIEW

File: micro_ros_setup/config/freertos/nucleo_f446ze/board.repos [hunk 1]
```diff
@@ -0,0 +1,5 @@
+repositories:
+  freertos_apps:
+    type: git
+    url: https://github.com/alsaibie/freertos_apps
```
Comment (pablogs9) [inline micro_ros_setup/config/freertos/nucleo_f446ze/board.repos:4]: Hello @alsaibie please the this URL to the correct link in the already merged repo: https://github.com/micro-ROS/freertos_apps

Comment (alsaibie) [reply micro_ros_setup/config/freertos/nucleo_f446ze/board.repos]: Done.

Comment (pablogs9) [reply micro_ros_setup/config/freertos/nucleo_f446ze/board.repos]: Thanks!

**Comment by pablogs9** [comment]
@alsaibie is it possible for you to sign commits for this PR?

**Comment by alsaibie** [comment]
OK. I hope I did the signature right.

**Comment by pablogs9** [comment]
Sorry, you didn't sign all of them. Maybe you can squash them all into one commit and do the signature in this one.

**Pull request merged**